Libraries/LibWeb/IndexedDB/Internal/Database.cpp

```diff
@@ -28,6 +28,16 @@ void Database::visit_edges(Visitor& visitor)
     visitor.visit(m_upgrade_transaction);
 }
 
+Vector<GC::Root<Database>> Database::for_key(StorageAPI::StorageKey const& key)
+{
+    Vector<GC::Root<Database>> databases;
+    for (auto const& database_mapping : m_databases.get(key).value_or({})) {
+        databases.append(database_mapping.value);
+    }
```

Comment:
Would it be possible to use `RootVector` instead and make use of the constructor that accepts a `ReadonlySpan`?